docs/source/references/schema.json

```diff
@@ -4845,10 +4844,6 @@
       {
         "name": "serialized",
         "type": "Vec<u8>"
-      },
-      {
-        "name": "blob_hash",
-        "type": "HashOf<WasmSmartContract>"
```

Comment:
this is good. I think that schema should also remove all references to `OptimizedExecutable`

Reply:
I'm not actually sure how it still gets there

Reply:
you should remove `OptimizedExecutable` from `data_model` and move it into `iroha_core`

Reply:
`OptimizedExecutable` should not be part of the public API. Everything in `data_model` is public API